client/gutenberg/extensions/subscriptions/edit.jsx

```diff
@@ -0,0 +1,70 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { Component } from '@wordpress/element';
+import { TextControl, Button, ToggleControl } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import { sprintf } from '@wordpress/i18n/build/index';
+import apiFetch from '@wordpress/api-fetch';
+
+class SubscriptionEdit extends Component {
+	render() {
+		const { attributes, className, isSelected, setAttributes } = this.props;
+		const { subscribe_placeholder, show_subscribers_total, subscriber_count_string } = attributes;
+
+		// Get the subscriber count so it is available right away if the user toggles the setting
+		this.get_subscriber_count();
```

Comment:
Would be good to move `this.get_subscriber_count()` to a lifecycle method especially since it seems to initiate an API call; for example `componentDidMount`?